packages/client3/components/design-system/form/form.tsx

```diff
@@ -75,11 +73,12 @@ export function Form<FormFields>({
         {children}
         {/* @ts-expect-error Same as above */}
         {errors[reservedErrorField] ? (
-          <HelperText className="mt-2">
-            {/* @ts-expect-error Same as above */}
-            Error: {errors[reservedErrorField].message}
-          </HelperText>
-        ) : null}
+          <></>
+        ) : // <HelperText className="mt-2">
+        //   {/* @ts-expect-error Same as above */}
+        //   Error: {errors[reservedErrorField].message}
+        // </HelperText>
+        null}
```

Comment:
Old component used by goldfinch. I commented instead of deleting for future references.